portfolio/src/main/webapp/index.html

```diff
@@ -103,17 +100,16 @@ <h2> People Who inspire me </h2>
         <button onclick="getPageComments();">Comments!</button>
         <div style="font-size:200%;" id="comments-container"></div>
         <p></p>
-      <body onload="initMap()">
-         <div id="map"></div>
+        <div id="map"></div>
         <script>
         </script>
-    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCim4kl_9Y7zCW1zeNnpgejSHmyTaxV37g&callback=initMap"
-    async defer></script>
+        <script src="https://maps.googleapis.com/maps/api/js?key=GOOGLE_APPLICATION_CREDENTIALS&callback=initMap"
```

Comment:
Does this actually work? OR does it just think it's the string "GOOGLE_APPLICATION_CREDENTIALS"

Reply:
Actually I do not think it works . I was just playing around with it and forgot to remove it because I push